server/mediapublic/mediapublic/views.py

```diff
@@ -594,8 +596,138 @@ def blogs_comment_by_id(request):
     id = request.matchdict['id']
     if not request.method in METHODS:
         resp = {'error': 'Method Not Allowed'}
+        resp, status = METHODS[request.method](request, id, cls)
         status = 405
     else:
-        resp, status = METHODS[request.method](request, id, cls)
+    request.response.status = status
+    return resp
+
+########### PLAYLISTS
+# [GET]
+@view_config(route_name='playlists', renderer='json')
+def station_playlists(request):
+    cls = Playlists
+    resp = []
+    if request.method == "GET":
+        resp = [i.to_dict() for i in cls.get_all()]
```

Comment:
I believe I had done ...

    items = cls.get_all()
    resp = [i.to_dict() for i in items]

... for all of the other `GET` methods.  Doesn't really matter, but if you think we should combine them, let's combine them everywhere.